authenticity-collector/is-authenticity-collector/src/identity/identity.service.ts

```diff
@@ -0,0 +1,60 @@
+import { BadRequestException, Injectable, Logger } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { firstValueFrom } from 'rxjs';
+import { InjectModel } from '@nestjs/mongoose';
+import { Identity, IdentityDocument } from './schemas/identity.schema';
+import { Model } from 'mongoose';
+import { IdentityDto } from './models/IdentityDto';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class IdentityService {
+    private readonly logger = new Logger(IdentityService.name);
```

Comment:
removed if logging is not recommended.

Reply:
it is recommended to log errors but not every step the api does, so please keep it for the errors

Reply:
ok. added them back. But I can inject the Logger if I'm defining a customer logger in a separate module and import this. I guess this will be a bit overengnieering for this service.

Reply:
ah ok yes then for now we keep it as it is without injecting